File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
    /*         l
                          r
        [2,10,30,30,30,30]
    */
public:
    int removeDuplicates(vector<int>& nums) {
        int left = 0;
        for(int right = 0; right < nums.size(); right++){
            if(nums[right] != nums[left])
                nums[++left] = nums[right];
        }

        return left +1;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        prevIdx = {}
        for i in range(len(nums)):
            if (target - nums[i]) in prevIdx:
                return [prevIdx[target - nums[i]], i]
            
            prevIdx[nums[i]] = i
        
        return []